### DISCUSSION TITLE
关于后端无法找到win32gui的bug

### DISCUSSION THREAD

**Discussion by nssssfw** [infstellar/python-git-program-launcher#38] [Q&A]
Traceback (most recent call last):
  File "genshin_assistant.py", line 6, in <module>
    import source.util
  File "E:\MAG\PGPL-2.2.0\repositories\GIA_Launcher_Download_Lib\source\util.py", line 3, in <module>
    import win32gui, win32process, psutil, ctypes, pickle, traceback
ModuleNotFoundError: No module named 'win32gui'
请按任意键继续. . .

重下后我的后端每次都会这样显示，我觉的我的电脑已经无法再运行这个软件了。
那一天我删了整个文件夹，因为我感觉有点问题，结果下载回来是就这样了。
重装过win32gui也没用，整个python删了重装也没用，这是怎么了😢？

**Comment by sanyueqi520** [comment]
我也是一样的问题

**Comment by nssssfw** [comment]
我最后也没解决，干脆重装了电脑治好了😉



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年12月11日 22:16 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [infstellar/python-git-program-launcher] 关于后端无法找到win32gui的bug (Discussion #38) |

我也是一样的问题
